src/common/ConsoleMessage.ts

```diff
@@ -16,20 +16,55 @@
 
 import { JSHandle } from './JSHandle';
 
-interface ConsoleMessageLocation {
+/**
+ * @public
+ */
+export interface ConsoleMessageLocation {
+  /**
+   * URL of the resource if known or `undefined` otherwise.
+   */
   url?: string;
+
+  /**
+   * 0-based line number in the resource if known or `undefined` otherwise.
+   */
   lineNumber?: number;
+
+  /**
+   * 0-based column number in the resource if known or `undefined` otherwise.
+   */
   columnNumber?: number;
 }
 
+// Prettier seems to struggle with the ConsoleMessageType declaration
+// so it is switched off just for that block.
```

Comment:
what do you mean by struggle? https://prettier.io/playground/#N4Igxg9gdgLgprEAuEcAeAHCAnGACGATwzjwGFoBnCAGzgFk5LKBDAczgBVjSBePAOQ0IbAXgA+ggCZwARgFdREwQEsoAMwhjJAuNmw5tggO4tsUNUvEAdKHnvSV2IwKlO0AWxouYLWXR9sFjA4FzA6MxdKX1wAcQN5DG1bB0Fosxh4iESKGhoWDEo4KRcEKSzElxZmPRgXDAN1FQCJFIcBBogmugBRKBLlAUh5WB8VDzg+gZ0ANz1ZCCKBAG5bEAAaEAgMGBUqZFAzA2MABTMESmQQFhpTQkvN2SCwAGs4GABlDGDLZBhseRwTYACxgXgA6sCVPBKN8Qh8LtCVDNoYQrmBmBsQGoirgTkE2B4WMh1DcipsAFaUNAAIWeb0+LAmABk1HASWSgSAqWgPpY6ABFeQQeAcmjkkDfbC4q6yPxwGhYhpqGDglRSGDA5AADgADJtOkVwUEMFcGkw9HMsQBHYXwfHbS4oaoAWigcGKxSx2DgtqccHx7CJYolRQ8Kj+AK5lH5cCFIvZSFJ4q5vlkao1WqQACZNv8WM1LBQPMGUEwAKxY+RFTh+J3JiUzQEASX6CE+YGwKh2AEF+h8iHQQ3AAL4joA seems OK

Reply:
Yeah I don't know. As per our offline conversation, it seems unhappy in the context of the file itself. Maybe a version difference?

Reply:
Yeah maybe! Let's leave this be for now. Not worth blocking.